doc/dev.rst

```diff
@@ -20,7 +20,7 @@ Setup development environment
 * Requires supported Python version
 * do setup under django-redshift-backend.git repository root as::
 
-    $ pip install -U pip setuptools wheel setuotools_scm
+    $ pip install -U pip setuptools wheel setuptools_scm
```

Comment:
Thanks! I've applied this line at https://github.com/jazzband/django-redshift-backend/commit/ff5c1cbef6b8aceee304d5b2cd396e44c451852d